packages/react-composites/src/composites/CallComposite/adapter/CallingSoundSubscriber.ts

```diff
@@ -14,8 +14,11 @@ import { isPhoneNumberIdentifier } from '@azure/communication-common';
 type CallingSoundsLoaded = {
   callEndedSound?: HTMLAudioElement;
   callRingingSound?: HTMLAudioElement;
+  callBusySound?: HTMLAudioElement;
 };
 
+const CALL_REJECTED_CODE = 603;
```

Comment:
can we have a central file that has all of the codes in it? `CallSDKCodeConstants.ts` or something

Reply:
This is a good idea, I will work with Miguel to find a place and put any Rooms codes in there too

Reply:
Yeah that's a good idea!